src/bin/cargo/commands/owner.rs

```diff
@@ -6,25 +6,68 @@ use cargo::util::auth::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg_quiet()
         .arg(Arg::new("crate").action(ArgAction::Set))
-        .arg(
-            multi_opt(
-                "add",
-                "LOGIN",
-                "Name of a user or team to invite as an owner",
-            )
-            .short('a'),
-        )
-        .arg(
-            multi_opt(
-                "remove",
-                "LOGIN",
-                "Name of a user or team to remove as an owner",
-            )
-            .short('r'),
+        .arg_required_else_help(true)
+        .override_usage(
+            "\
+       cargo owner [OPTIONS] add    OWNER_NAME <CRATE_NAME>
+       cargo owner [OPTIONS] remove OWNER_NAME <CRATE_NAME>
+       cargo owner [OPTIONS] list   <CRATE_NAME>",
         )
-        .arg(flag("list", "List owners of a crate").short('l'))
+        .arg_quiet()
+        .subcommands([
+            Command::new("add")
+                .about("Name of a user or team to invite as an owner")
+                .override_usage(
+                    "\
+                   cargo owner [OPTIONS] add [OWNER_NAME] <CRATE_NAME>",
+                )
+                .arg_quiet()
+                .args([
+                    Arg::new("ownername")
+                        .action(ArgAction::Set)
+                        .required(true)
+                        .num_args(1)
```

Comment:
`num_args(1)` is the default and shouldn't be needed

Reply:
It's cancelled.